Server/src/controllers/GraphsController.ts

```diff
@@ -84,16 +97,17 @@ export class GraphsController {
      * @param response 
      * An object containing a response: Response
      */
-    async createRequestForDeletingSavedGraph(request: Request, response: Response) {
-        let requestParam = request.params.deleteSavedGraph;
+    async createRequestForDeletingGraph(request: Request, response: Response) {
+        let userId = request.body.user.account_id
+        let requestParam = request.params.graphStateId;
         let graphId: number = +requestParam;

```

Comment:
whatsup with the '+'requestParam?

Reply:
Part of number casting. I could rewrite the line as follows if that would be clearer:
let graphId = Number(requestParam);